docs/configuration.rst

```diff
@@ -14,6 +14,8 @@ Will be used if [tool.rope] is configured.
 
     [tool.rope]
     split_imports = true
+    [tool.rope.autoimport]
+    underlined = false
```

Comment:
I'd much prefer to keep a flat namespace here for the config keys. The vast majority of users are going to only have a small amount of config, creating multiple config sections just for a handful of configuration seems overkill.

Reply:
IIRC, toml allows [dotted keys](https://toml.io/en/v1.0.0). Maybe we can have our cake and eat it too, by recommending the dotted form in the examples section instead of the separate section:

```toml
[tool.rope]
autoimport.underlined = true
```